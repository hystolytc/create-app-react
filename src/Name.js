import React from 'react'

const Name = () => (
  <div>This is Name component</div>
)

export default Name