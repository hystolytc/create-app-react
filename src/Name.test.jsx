import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render, screen } from '@testing-library/react'
import Name from './Name'

const describeComponent = '<Name /> test case'

describe(describeComponent, () => {
  const { container } = render(<Name />)

  test(`Render ${describeComponent}`, () => {
    expect(container).toMatchSnapshot()
  })
})