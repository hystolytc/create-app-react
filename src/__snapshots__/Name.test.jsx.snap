// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Name /> test case Render <Name /> test case 1`] = `
<div>
  <div>
    This is Name component
  </div>
</div>
`;
